test/unit/controllers/actions/JourneyRefinerSpec.scala

```diff
@@ -43,6 +43,7 @@ class JourneyRefinerSpec extends WordSpec with MustMatchers with MockitoSugar wi
   private val request = AuthenticatedRequest(FakeRequest(), user)
   private val answers = AssociateUcrAnswers()
   private val cache = Cache("eori", answers)
+  private val redirectResult = Results.Redirect(controllers.routes.ChoiceController.displayChoiceForm())
```

Comment:
Nit: In our projects we usually don't write `Result` before `Redirect`

Reply:
The line does not compile with error `not found: value Redirect` when `Result.Redirect` is  not used.